mavsdk/drone.py

```diff
@@ -0,0 +1,129 @@
+        if self._mavsdk_server_address is None:
+            self._mavsdk_server_address = 'localhost'
+            self._start_mavsdk_server(drone_address)
+
+        await self._init_plugins(self._mavsdk_server_address, self._port)
+
+    async def _init_plugins(self, host, port):
+        plugin_manager = await AsyncPluginManager.create(host=host, port=port)
+
+        for plugin in self._core_plugins:
+            self._plugins[plugin.lower()] = globals()[plugin](plugin_manager)
+
+    @property
+    def action(self) -> Action:
+        if "action" not in self._plugins:
+            raise RuntimeError("Action plugin has not been initialized! Did you run `Drone.connect()`?")
+        return self._plugins["action"]
+
+    @property
+    def calibration(self) -> Calibration:
+        if "calibration" not in self._plugins:
+            raise RuntimeError("Calibration plugin has not been initialized! Did you run `Drone.connect()`?")
+        return self._plugins["calibration"]
+
+    @property
+    def camera(self) -> Camera:
+        if "camera" not in self._plugins:
+            raise RuntimeError("Camera plugin has not been initialized! Did you run `Drone.connect()`?")
+        return self._plugins["camera"]
+
+    @property
+    def core(self) -> Core:
+        if "core" not in self._plugins:
+            raise RuntimeError("Core plugin has not been initialized! Did you run `Drone.connect()`?")
+        return self._plugins["core"]
+
+    @property
+    def gimbal(self) -> Gimbal:
+        if "gimbal" not in self._plugins:
+            raise RuntimeError("Gimbal plugin has not been initialized! Did you run `Drone.connect()`?")
+        return self._plugins["gimbal"]
+
+    @property
+    def info(self) -> Info:
+        if "info" not in self._plugins:
+            raise RuntimeError("Info plugin has not been initialized! Did you run `Drone.connect()`?")
+        return self._plugins["info"]
+
+    @property
+    def mission(self) -> Mission:
+        if "mission" not in self._plugins:
+            raise RuntimeError("Mission plugin has not been initialized! Did you run `Drone.connect()`?")
+        return self._plugins["mission"]
+
+    @property
+    def param(self) -> Param:
+        if "param" not in self._plugins:
+            raise RuntimeError("Param plugin has not been initialized! Did you run `Drone.connect()`?")
+        return self._plugins["param"]
+
+    @property
+    def offboard(self) -> Offboard:
+        if "offboard" not in self._plugins:
+            raise RuntimeError("Offboard plugin has not been initialized! Did you run `Drone.connect()`?")
+        return self._plugins["offboard"]
+
+    @property
+    def telemetry(self) -> Telemetry:
+        if "telemetry" not in self._plugins:
+            raise RuntimeError("Telemetry plugin has not been initialized! Did you run `Drone.connect()`?")
+        return self._plugins["telemetry"]
+
+    @staticmethod
+    def _start_mavsdk_server(drone_address=None):
```

Comment:
Why `drone_address`? What about `mavlink_address`?

Reply:
That was suggested in #86. I really don't mind. Should I go for `mavlink_address`? @hamishwillee any opinion?

Reply:
It's now `system_address`